thorlcr/graph/thgraph.hpp

```diff
@@ -65,6 +65,7 @@
 
 #include "thorplugin.hpp"
 #include "jstats.h"
+#include "dautils.hpp"
```

Comment:
probably unnecessary change? and only reason this file is changing?

Reply:
IFileReadPropertiesUpdater and createFileReadPropertiesUpdater is defined in dautils.hpp.  IFileReadPropertiesUpdater and createFileReadPropertiesUpdater is used in thgraphmaster.cpp, thgraphmaster.ipp, eclgraph.ipp and eclgraph.cpp.   Adding the include file in thgraph.hpp allows IFileReadPropertiesUpdater and createFileReadPropertiesUpdater to be used in all these locations.

Reply:
thgraph.* is common to workers+manager.
As these were introduced and used by thgraphmaster.* the include should really go in thgraphmaster.ipp too.

Reply:
IFileReadPropertiesUpdater  is also used by thgraph.cpp and eclagent.ipp.  It seems thgraph.hpp covers all the cases for master, worker and hthor/eclagent.

I'm not sure why thgraphmaster.ipp should also include dautils- thgraphmaster.ipp includes thgraph.hpp (which include dautils.hpp)

Reply:
> is also used by thgraph.cpp

er. where? thgraph.cpp did not change